prep-html-syntax/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+
+<head> <title> Prep-Html-Syntax </title>
+</head>
+<body>
+
+  <h1 style="color: darkred;"> My favorite Star Wars character is Darth Maul!</h1>
+  <img src="https://bbts1.azureedge.net/images/p/full/2020/10/71a2f12c-99b9-46e5-84c2-a09231879526.jpg" alt="Darth Maul">
+  <p1 style="color: darkseagreen;"> <br> Darth Maul is my favorite Star Wars character because he looks so cool. Also Sith GANG.</p1>
```

Comment:
Hi Andrew! Heads up, `<p1>` is not a valid HTML tag. Try `<p>` instead. That way, you won't need the `<br>`.